Exercises-1-Basic/src/exercises/Number5Shape2.java

```diff
@@ -0,0 +1,19 @@
+package exercises;
+/* ii)
+*
+**
+***
+****
+*****
+ */
+public class Number6Shape2 {
```

Comment:
after I changed file's name like you said, all your comments are disappeared :((